modules/aeternity/utils/tipTopicUtil.js

```diff
@@ -40,20 +41,25 @@ const getTipTopics = tips => { // TODO move to db
   }, {});
 
   const maxCount = Math.max(...Object.values(topics).map(x => x.count));
+  const maxAmount = Math.max(...Object.values(topics).map(x => new BigNumber(x.amount).toNumber()));
 
   const sortedTopic = Object.entries(topics).map(([topic, data]) => {
     const topicData = data;
     topicData.avgScore = data.totalScore / data.count;
     topicData.countScore = data.count / maxCount;
+    topicData.amountScore = data.amount / maxAmount;
 
     topicData.score = topicData.avgScore * avgTipScoreWeight
-      + topicData.countScore * countScoreWeight;
+      + topicData.countScore * countScoreWeight
+      + topicData.amountScore * amountScoreWeight;
+
+    if (topicData.amountScore === 0) topicData.score = 0
```

Comment:
POST_WITHOUT_TIPS are never in trending then?

Reply:
As far as I did understand the request was that no 0 amount are shown as trending, POST_WITHOUT_TIPS can trend if the hashtag is also used in tips with amount